login.html

```diff
@@ -67,7 +113,7 @@
     <script src="vendor/jquery-easing/jquery.easing.min.js"></script>
     <script src="js/egna-legna.js"></script>
     <script>
-      $(".login-button").click(function(){
+      $(".login-button").click(you must eneter both user name and password(){
```

Comment:
This won't work as expected, that requires passing anonymous function to be passed as parameter to the click function.